main.tf

```diff
@@ -252,6 +252,12 @@ resource "aws_lb_listener_rule" "unauthenticated_access_for_cidr_blocks" {
     target_group_arn = module.alb.target_group_arns[0]
   }
 
+  condition {
```

Comment:
while this may work (haven't fully tested the functionality) it does impose a change for all users. if we instead turned this into a dynamic block, we could instead have a no-op change for most users who don't opt into using an access path pattern

Reply:
right, `dynamic block` is better. will change it soon.

Reply:
fix in ea06460